cockatrice/src/settings/cache_settings.cpp

```diff
@@ -537,6 +538,12 @@ void SettingsCache::setTapAnimation(QT_STATE_CHANGED_T _tapAnimation)
     settings->setValue("cards/tapanimation", tapAnimation);
 }
 
+void SettingsCache::setOpenDeckInNewTab(QT_STATE_CHANGED_T _openDeckInNewTab)
+{
+    openDeckInNewTab = static_cast<bool>(_openDeckInNewTab);
+    settings->setValue("editor/open_deck_in_new_tab", _openDeckInNewTab);
```

Comment:
this should be `openDeckInNewTab` (without the underscore)

Reply:
Unfortunately it's not uniform in the file.

Like
`settings->setValue("cards/tapanimation", tapAnimation);`
or
`settings->setValue("table/invert_vertical", invertVerticalCoordinate);`